app/config/config.yml

```diff
@@ -73,12 +73,14 @@ assetic:
 
 prooph_service_bus:
   command_buses:
-    main:
+    todo:
+      plugins:
+        - 'Prooph\EventStoreBusBridge\TransactionManager'
```

Comment:
@sandrokeil note: it works now, but we should not use class names as service ids because it is uncommon in the symfony world I guess.